### PR TITLE
HTTP/2 on MDN

### PR DESCRIPTION
**Pull request by stephaniehobson**
I can't test these locally can someone make sure it displays properly please?

### CODE REVIEW

File: _posts/2017-06-26-http2.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+We enabled HTTP/2 on MDN's CDN.
+
+We didn't do anything to optimize for HTTP/2, we just enabled it.
+
+We're seeing performance improvements.
+
+<!--more-->
+
+Why is this worth a blog post?
```
Comment (jwhitlock) [inline _posts/2017-06-26-http2.md]: Consider stating the conclusion instead, like "HTTP/2 isn't scary" or "You don't have to be perfect to use HTTP/2"

File: _posts/2017-06-26-http2.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+![Waterfall showing DNS lookup for only first asset requested.]({{ site.baseurl }}/public/images/2017-06-26-http2/waterfall_after.png)
+
+Some numbers
+===
+
+Overall, our tests don't show a huge improvement in page load speed but there
+are small improvements across the board, and a real impact for users located far
+away from our servers. (Hi Australia and China!)
+
+| Service | Location | Browser | HTTP/1.1 | HTTP/2 | Change |
```
Comment (jwhitlock) [inline _posts/2017-06-26-http2.md]: I was mentally calculating percents, like 13% speedup for Dallas, and a 44% improvement (!) for Austrailia in GA. Does it make sense to add a percent column?

File: _posts/2017-06-26-http2.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+| Google Analytics | All | Firefox | 3.99s | 3.71s | 0.28s |
+| Google Analytics | Australia | All | 3.01s | 1.69s | 1.32s |
+| Google Analytics | China | All | 8.10s | 6.69s | 1.41s |
+
+I tried to segment our users in Google Analytics to make sure we did not have
+a negative impact on users relying on HTTP/1.1 and... couldn't find enough
+users to draw any conclusions. MDN is lucky like that. (It's possible the IE11
+test in the table above is on Windows 7 and does not support HTTP/2, but
+WebPageTest doesn't identify the OS.) In theory, older browsers
+should not be effected because the protocol falls back to HTTP 1.1.
```
Comment (jwhitlock) [inline _posts/2017-06-26-http2.md]: effected -> affected

File: _posts/2017-06-26-http2.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+Which suggest that enabling HTTP/2 on a site optimized for HTTP/1.1 could result
+in a slower site.
+
+A better way to interpret those quotes is:
+
+> If you optimize for HTTP/1.1 and turn on HTTP/2 your site will not be as fast
+> as it could be - but it might still be faster!
+
+On MDN we concatenate a lot of our files but we don't concatenate all of them.
+For example, our article pages we have 9 different files coming from our CDN. I
```
Comment (jwhitlock) [inline _posts/2017-06-26-http2.md]: I think this should be "**on** our article pages we have 9 different files" or "our article pages ~~we~~ have 9 different files..."

File: _posts/2017-06-26-http2.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Next
+===
+
+**Get our pages into data centres around the world.**
+
+This involves changing our hosting services, not a small task, and changing our
+pages to serve the same content to all logged out users.  
+
+
+**Decreasing asset size by removing support for older browsers.**
```
Comment (jwhitlock) [inline _posts/2017-06-26-http2.md]: I think "Decrease" instead of "Decreasing"

**Comment by stephaniehobson** [comment]
Grammar improved, heading improved, not doing maths :P

**Comment by jwhitlock** [comment]
Thank, let's publish!

**Comment by jwhitlock** [comment]
I also can't get Jekyll to render blog posts with future publish dates.  My solution was to change the date in the file to a past date, and then review that.  Sorry I didn't give you that tip last week!

**Comment by pmac** [comment]
Or you can use the "docker-run.sh" script in the repo. It uses a docker
container to run the Jekyll site and uses the "--drafts" switch to include
drafts in the output.

**Comment by stephaniehobson** [comment]
Neither of those fix the fact that I can't run Ruby :P

**Comment by pmac** [comment]
The script does actually, because it installs and runs ruby in the docker image only. So if you have Docker installed it should "Just Work".

**Comment by jwhitlock** [comment]
I wasn't able to get drafts working. If you can, please update the README.

**Comment by jwhitlock** [comment]
Got drafts working.  I was trying to use ``_drafts/2017-06-29-hello.md`` as the filename, but it should not have the embedded date.  ``_drafts/hello.md`` works as documented.  README update inbound!

**Pull request merged**